apps/web/src/pages/Applications/ApplicationSkillsPage/components/SkillTree.tsx

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import {
+  Button,
+  CardBasic,
+  Heading,
+  TreeView,
+  Well,
+  HeadingLevel,
+  Accordion,
+} from "@gc-digital-talent/ui";
+import { getLocalizedName } from "@gc-digital-talent/i18n";
+
+import { Experience, Skill } from "~/api/generated";
+import { getExperienceSkills } from "~/utils/skillUtils";
+import ExperienceAccordion from "~/components/ExperienceAccordion/ExperienceAccordion";
+
+import SkillFormDialog from "./SkillFormDialog";
+
+interface SkillTreeProps {
+  skill: Skill;
+  experiences: Experience[];
+  headingAs?: HeadingLevel;
+  showDisclaimer?: boolean;
+}
+
+const SkillTree = ({
```

Comment:
Would it be worth it to filter out the non-applicable experience skill records for this view, it will be reused here and admin-side I believe. Otherwise, may look a bit cluttered? At least it probably shouldn't show skills not applicable to the recruitment in question.

![image](https://user-images.githubusercontent.com/40485260/236519559-759664aa-c04e-4045-990a-a423f626e5f3.png)

![image](https://user-images.githubusercontent.com/40485260/236519641-35cba04d-0f2e-4b06-9194-a2d518f899fe.png)

Reply:
Good idea, updated in [filter out unrelated skills](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6518/commits/3f92c6642881041b1c0d8ec127dbf7c275b0436c)